benchmarks/models.py

```diff
@@ -0,0 +1,112 @@
+import torch
+from torch.nn import Linear, ReLU, CrossEntropyLoss, Sequential, Conv2d, MaxPool2d, Softmax, Module, BatchNorm2d
+
+
+class Model_0(Module):
+    
+
+    def __init__(self):
+        super(Model_0, self).__init__()
```

Comment:
In python `3.X` you don't need to put anything in `super`
```suggestion
        super().__init__()
```